index.js

```diff
@@ -109,11 +93,15 @@ async function captureSerializedDOM(driver, options) {
     /* eslint-disable-next-line no-undef */
     domSnapshot: PercyDOM.serialize(options)
   }), options);
-  domSnapshot.cookies = await driver.manage().getCookies();
+  /* istanbul ignore next: no instrumenting injected code */
+  domSnapshot.cookies = await driver.manage().getCookies() || {};
```

Comment:
```suggestion
  domSnapshot.cookies = await driver.manage().getCookies() || [];
```
cookies are list of objects

Reply:
can remove this it doesn't need. will remove that check.

Reply:
done.